Engineering/PracticiesAndPrinciples.md

```diff
@@ -18,6 +18,8 @@ While working on the project we try to stick with best practices in architecture
 * Remember to tether to ‘Done’ for a backlog item. Our goal is not to write code, it is to ship incremental customer value. Our job is likely not done until our code is in production adding that value.
 * We all own our code and each one of us has an obligation to make all parts of the system great
 * Keeping backlog item details up to date is how you communicate the state of things with the rest of your team. Discipline around updating status, assignment, comments etc is essential
+* Automate agile workflows using tools like Standup bots (in Microsoft Teams or other communication tools) to "give back" more time to developers for development.
```

Comment:
I haven't seen any teams using standup bot (though it looks nice), but as a general guidance I don't know if it will work. Standup Bot looks like it doesn't work in Teams and it costs money, which can be a problem for many engagements.